test/TestAppIsolated/Program.cs

```diff
@@ -10,6 +10,6 @@
 //     .AddApplicationInsightsTelemetryWorkerService()
 //     .ConfigureFunctionsApplicationInsights();
 
-builder.EnableMcpToolMetadata();
```

Comment:
Removed this as we're registered the decorator in startup, but we can also include this in `ConfigureMcpExtension` instead